neo/Ledger/Blockchain.cs

```diff
@@ -727,6 +729,49 @@ private void UpdateCurrentSnapshot()
         {
             Interlocked.Exchange(ref currentSnapshot, GetSnapshot())?.Dispose();
         }
+
+        //save Native Contract
+        void RegNativeContract()
+        {
+            using (Snapshot snapshot = GetSnapshot())
+            {
+                foreach(var c in SmartContract.NativeContract.NativeContractList.Contracts.Values)
+                {
+                    RegNativeContractWithDB(snapshot, c);
+                }
+                snapshot.Commit();
+            }
+        }
+        void RegNativeContractWithDB(Snapshot snapshot, SmartContract.NativeContract.INativeContract nativecontract)
```

Comment:
Reg means Register?
Please use complete name here @lightszero for better clarity: RegisterNativeContractWithDB